identityClient/api.go

```diff
@@ -1190,3 +1190,39 @@ type FederatedIdentityKeyCheckRequest struct {
 	ClientID                        uuid.UUID `json:"client_id"`
 	PasswordDerivedPublicSigningKey string    `json:"password_derived_public_signing_key"`
 }
+
+// AccessControlPolicyRequest wraps the values required to request Access Control Policy for an Application.
```

Comment:
I think the following is more accurate.
`AccessControlPolicyRequest wraps the values required to enable or disable an Access Control Policy for an application.`

Reply:
Will need to update comments for a lot of the structs and methods!